editor/src/components/canvas/ui-jsx-canvas-renderer/scene-root.tsx

```diff
@@ -96,10 +108,20 @@ function useGetValidTemplatePaths(
   topLevelElementName: string | null,
   scenePath: ScenePath,
 ): Array<InstancePath> {
-  const topLevelElements = useContextSelector(RerenderUtopiaContext, (c) => c.topLevelElements)
+  const uiFilePath = useContextSelector(UtopiaProjectContext, (c) => c.openStoryboardFilePathKILLME)
+
+  const topLevelElements = useGetTopLevelElements(uiFilePath ?? '')
```

Comment:
`?? ''` <- This feels a bit wrong, surely the hook should handle the nullable case?